### PR TITLE
[google_sign_in_web] README.md typo

### PR DESCRIPTION
**Pull request by Jerinji2016**
This PR has changes to a typo in the README.md in [google_sign_in_web](https://github.com/flutter/packages/tree/main/packages/google_sign_in/google_sign_in_web)

List of Issues fixed:
- Typo in README.md of [google_sign_in_web](https://github.com/flutter/packages/tree/main/packages/google_sign_in/google_sign_in_web)

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I [linked to at least one issue that this PR fixes] in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[linked to at least one issue that this PR fixes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Comment by stuartmorgan** [comment]
Thanks for the contribution! You’ve checked boxes in the PR checklist above that are not reflected in this PR, so I’m assuming this is a work in progress and am marking it as a Draft. Please review the checklist, updating the PR as appropriate, and when the checklist reflects the state of the PR as posted please feel free to mark it as ready for review.

**Comment by Jerinji2016** [comment]
@ditman 
Requested changes are done

File: packages/google_sign_in/google_sign_in_web/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 0.12.4+1
+* Fixed README.md typo
```
Comment (stuartmorgan) [inline packages/google_sign_in/google_sign_in_web/CHANGELOG.md:2]: Please follow the changeliog style guide linked from the PR checklist.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

File: packages/google_sign_in/google_sign_in_web/CHANGELOG.md (hunk 1 above)
Comment (Jerinji2016) [reply packages/google_sign_in/google_sign_in_web/CHANGELOG.md:2]: I think I need to change version to *NEXT*.  
Should I revert the version change in pubspec too?

Comment (stuartmorgan) [reply packages/google_sign_in/google_sign_in_web/CHANGELOG.md:2]: I recommend reading the FAQ section of the linked documentation about READMEs and version updates. Also, you haven't addressed the CHANGELOG message style issues raised in the comment you are replying to.

Please read the documentation linked from the PR checklist; the reason we have the checklist is to provide guidance and links to relevant documentation.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

File: packages/google_sign_in/google_sign_in_web/CHANGELOG.md (hunk 1 above)
Comment (stuartmorgan) [reply packages/google_sign_in/google_sign_in_web/CHANGELOG.md:3]: This still doesn't use the correct verb form, and still doesn't have a period, so none of the style issues have been resolved.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]
This PR is still not following a lot of our process. The process is here for a reason; please don't skip it.

> _Replace this paragraph with a description of what this PR is changing or adding, and why. Consider including before/after screenshots._

This is still placeholder text.

> * [x]  The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`

This box is checked despite this step not having been done.

> * [x]  I [linked to at least one issue that this PR fixes](https://github.com/flutter/flutter/wiki/Tree-hygiene#overview) in the description above.

Why is this box checked?

> * [x]  I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style).

This style guide is still not being followed in the PR, as noted below.

**Comment by Jerinji2016** [comment]
@stuartmorgan 
Pardon for the annoying review requests. Have added the changes as asked

**Review by stuartmorgan — APPROVED** [review]
LGTM, thanks!

**Review by ditman — APPROVED** [review]
Thanks!!

**Pull request merged**